sources/packages/backend/libs/test-utils/src/factories/student-file-uploads.ts

```diff
@@ -0,0 +1,48 @@
+import * as faker from "faker";
+import { FileOriginType, Student, StudentFile, User } from "@sims/sims-db";
+import { DataSource } from "typeorm";
+import { createFakeStudent } from "./student";
+
+/**
+ * Create fake student file upload object.
+ * @param student entity.
+ * @returns created studentFile object.
+ */
+export function createFakeStudentFileUpload(
+  relations?: {
+    student?: Student;
+    creator?: User;
+  },
+  options?: {
+    fileOrigin?: FileOriginType;
+  },
+): StudentFile {
+  const studentFile = new StudentFile();
+  studentFile.fileName = faker.system.fileName();
+  studentFile.uniqueFileName =
+    studentFile.fileName + faker.random.uuid() + "." + faker.system.fileType();
+  studentFile.groupName = "Ministry communications";
+  studentFile.mimeType = faker.system.mimeType();
+  studentFile.fileContent = Buffer.from(faker.random.words(50), "utf-8");
+  studentFile.student = relations?.student ?? createFakeStudent();
+  studentFile.creator = { id: relations?.creator?.id } as User;
```

Comment:
It can be simplified to `studentFile.creator = creator;`, unless I am missing something.